import "proto_struct.proto";

message ClientSendData
{
	optional string SenderName=1;
	optional string TalkMsg = 2;				//聊天信息(25中文字符)
}

message ServerSendData
{
	optional string Result			= 1;		//结果
	optional string NickName		= 2;		//角色名称
	optional string TalkMsg			= 3;		//聊天信息
	optional string TalkTime		= 4;		//聊天时间
}

message csRegisterAccount
{
	optional string accountName=1;
	optional string password=2;
	optional string repeatPassword=3;
}

message scRegisterAccount
{
	optional bool result=1;
	
}

message csLoginGame
{
	optional string accountName=1;
	optional string password=2;
}

message scLoginGame
{
	optional bool result=1;
}

message csServerList
{
	
}

message scServerList
{
	repeated struct_serverList ServerList=1;
}

message csEnterGame
{
	optional int32 ServerId=1;
}

message scEnterGame
{

	optional int32 ServerId=1;
}



